addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -135,47 +135,58 @@ def _check_peppol_fields(self):
 
     @api.depends('country_code')
     def _compute_ubl_cii_format(self):
+        format_mapping = {
+            'DE': 'xrechnung',
+            'AU': 'ubl_a_nz',
+            'NZ': 'ubl_a_nz',
+            'NL': 'nlcius',
+            'FR': 'facturx',
+            'SG': 'ubl_sg',
+        }
         for partner in self:
-            if partner.country_code == 'DE':
-                partner.ubl_cii_format = 'xrechnung'
-            elif partner.country_code in ('AU', 'NZ'):
-                partner.ubl_cii_format = 'ubl_a_nz'
-            elif partner.country_code == 'NL':
-                partner.ubl_cii_format = 'nlcius'
-            elif partner.country_code == 'FR':
-                partner.ubl_cii_format = 'facturx'
-            elif partner.country_code == 'SG':
-                partner.ubl_cii_format = 'ubl_sg'
-            elif partner.country_code in EAS_MAPPING:
+            country_code = partner._get_peppol_country_code()
```

Comment:
so you need to add the dependency to vat, siret etc to `_compute_ubl_cii_format` :see_no_evil: